skyvern/forge/sdk/api/llm/config_registry.py

```diff
@@ -72,3 +72,11 @@ def get_config(cls, llm_key: str) -> LLMRouterConfig | LLMConfig:
             True,
         ),
     )
+if SettingsManager.get_settings().ENABLE_GEMINI:
+    LLMConfigRegistry.register_config(
+        "GEMINI_GPT4V", 
```

Comment:
```suggestion
        "GOOGLE_GEMINI_PRO", 
```

Reply:
![Screenshot from 2024-04-04 09-52-16](https://github.com/Skyvern-AI/skyvern/assets/72747883/ee94a695-e0dc-4c69-b3b5-837102bea67c)

Failing When i run 
`precommit run all-files`

Reply:
are you running it within an active poetry shell?

first run, poetry shell, then run the precommit command. let me know how it goes

Reply:
![image](https://github.com/Skyvern-AI/skyvern/assets/72747883/c5b5bd67-1c71-4b75-863f-02e6bc1e0ac0)

Reply:
![image](https://github.com/Skyvern-AI/skyvern/assets/72747883/f4ee5fe9-2c2b-43c7-abb5-0f7f561f81dd)
`got this error when running ./run_skyvern.sh`
is there any code changes others than above mentioned files.

Reply:
Yes, we have a check that makes sure at least one provider is enabled. You need to update here as well:

https://github.com/Skyvern-AI/skyvern/blob/c538523c88ded6457b6a6f4c7b1e4a31756b2aef/skyvern/forge/sdk/api/llm/config_registry.py#L46C1-L54C35